addons/website_forum/models/forum.py

```diff
@@ -934,6 +973,75 @@ def go_to_website(self):
             'url': self._compute_website_url(),
         }
 
+    @api.model
+    def _search_get_detail(self, website, order, options):
+        with_description = options['displayDescription']
+        with_date = options['displayDetail']
+        search_fields = ['name']
+        fetch_fields = ['id', 'name']
+        mapping = {
+            'name': {'name': 'name', 'type': 'text', 'match': True},
+            'website_url': {'name': 'website_url', 'type': 'text'},
+        }
+
+        domain = website.website_domain()
+        domain += [('parent_id', '=', False), ('state', '=', 'active'), ('can_view', '=', True)]
```

Comment:
Don't modify this pr from that but... please avoid using `+=`, its semantics are a bit shit (it often unexpectedly works in-place) and especially when appending a single element it's more expensive than, well, `append`.

Prefer explicit `append` and `extend`, and if you want to perform a copying concatenation, use separate assignment and addition.